lib/src/formatter.dart

```diff
@@ -169,7 +169,7 @@ class SimpleFormatter implements ReportFormatter {
     var codes = stats.keys.toList()..sort();
     var largestCountGuess = 8;
     var longest =
-        codes.fold(0, (int prev, element) => max(prev, element.length));
+        codes.fold(0, (int prev, element) => max/*<num>*/(prev, element.length));
```

Comment:
That would be because we don't know that `element` is a String, which means we lost type information somewhere. We should identify where and see whether we can do a better job of preserving the information.